daemons/Makefile

```diff
@@ -19,6 +19,14 @@ dss-sync dss-index dss-s3-copy-sfn dss-s3-copy-write-metadata-sfn dss-checkout-s
 	    $(DSS_HOME)/scripts/deploy_checkout_lifecycle.py; \
 	fi
 
+dss-visitation:
+	git clean -df $@/domovoilib $@/vendor
+	shopt -s nullglob; for wheel in $@/vendor.in/*/*.whl; do unzip -q -o -d $@/vendor $$wheel; done
+	cp -R ../dss ../dss-api.yml $@/domovoilib
+	cp "$(GOOGLE_APPLICATION_CREDENTIALS)" $@/domovoilib/gcp-credentials.json
+	./build_deploy_config.sh $@ $(DSS_DEPLOYMENT_STAGE)
+	cd $@; domovoi deploy --stage $(DSS_DEPLOYMENT_STAGE)
```

Comment:
Why is this target duplicated from the others? Can you roll it back in with the other Makefile targets?

Reply:
Rolled back!